### ISSUE TITLE
Question about label propagation task

### ISSUE THREAD

**Issue by Un1Lee** [Harry-Zhi/semantic_nerf#30]
Thanks for your publish code! When I run your code about label propagation task, I found that the semantic loss is nan because gt_label almost is 0 after single-click. You mention that "we do not apply any loss on the void regions hence the network is able to predict arbitrary classes without penalty (though in fact it tend to predict some reasonable classes based on the similarity in appearance or geometry). And the void region also does not contribute to the evaluation metrics" in #3 . 
So it is easy-to-understand why the semantic loss is nan. However,  in this case, how can the network train to the effect that you demonstrate? What I should do?

**Comment by Un1Lee** [comment]
hello, the author told me some details and finally I solve this problem by set 'redution' parameter:
crossentropy_loss = nn.CrossEntropyLoss(ignore_index=-1, reduction='sum')
maybe torch change its funtion in different version, now it will return 0 instead of nan.
I think it is beaceuse when its reduction is mean, it will divided by weight which is 0 and return nan.
